src/components/layout.less

```diff
@@ -16,6 +16,15 @@
   -webkit-overflow-scrolling: touch !important;
 }
 
+
+@media (max-width: 991px) {
+  .sc-list {
+    height: auto;
```

Comment:
!import is necessary, because of the way `semantic`'s styles have precedence; would rather keep all overrides `!important` in here.